agent.tf

```diff
@@ -60,12 +60,7 @@ resource "google_compute_instance" "default" {
 
       # run the actual agent
       "cd agent",
-      "export VSTS_AGENT_INPUT_URL=https://dev.azure.com/${var.pipelines_org}",
-      "export VSTS_AGENT_INPUT_AUTH=pat",
-      "export VSTS_AGENT_INPUT_TOKEN=${var.pipelines_agent_pat}",
```

Comment:
let's keep the token as an environment variable. This is so that if there's anything monitoring what's running on the agent, they don't get to see the PAT. See [this discussion](https://unix.stackexchange.com/q/439497) for context :)

Reply:
Done. In my project I've been using the Ansible provisioner and specifically turned off logging for that step with:

```
no_log: True
```